app/validators/preprint-form-validator.js

```diff
@@ -0,0 +1,38 @@
+import { validator, buildValidations } from 'ember-cp-validations';
+
+const Validations = buildValidations({
+    title: {
+        description: 'Title',
+        validators: [
+            validator('presence', true),
+            validator('length', {
+                // minimum length for title?
+                max: 300,
```

Comment:
In this case, please do. Front-end validation is a way of giving the user immediate feedback for better user experience- we don't want to give the user false confidence if the backend wouldn't accept that value.